en/apispec/openapi.json

```diff
@@ -3057,311 +2530,3053 @@
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "headers": {
+              "Qtl-Request-Id": {
+                "schema": {
+                  "type": "string"
+                },
+                "description": "A string automatically generated by the API server to identify each request."
+              }
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "code": {
+                      "type": "string",
+                      "description": "InternalError"
                     },
-                    {
-                      "title": "RequestExpired",
-                      "properties": {
-                        "code": {
-                          "type": "string",
-                          "description": "RequestExpired"
-                        },
-                        "message": {
-                          "type": "string",
-                          "description": "Request has expired. Timestamp date is xxx."
-                        },
-                        "status": {
-                          "$ref": "#/components/schemas/response.status"
-                        }
-                      }
+                    "message": {
+                      "type": "string",
+                      "description": "We encountered an internal error. Please try again."
                     },
-                    {
-                      "title": "MissingArgument",
-                      "properties": {
-                        "code": {
-                          "type": "string",
-                          "description": "MissingArgument"
-                        },
-                        "message": {
-                          "type": "string",
-                          "description": "Required parameter is missing."
-                        },
-                        "status": {
-                          "$ref": "#/components/schemas/response.status"
+                    "status": {
+                      "$ref": "#/components/schemas/response.status"
+                    }
+                  }
+                },
+                "examples": {
+                  "InternalError": {
+                    "value": {
+                      "code": "InternalError",
+                      "message": "We encountered an internal error. Please try again.",
+                      "status": 0
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "operationId": "get-report-requests",
+        "description": "The API is to query the total request count for all of customer’s transports.  Parameters allow you to specify a time period, an interval type.",
```

Comment:
"This API returns the total number of requests for all of a customer's transports."